onnxruntime/core/framework/execution_frame.cc

```diff
@@ -42,6 +42,16 @@ OrtValue* IExecutionFrame::GetMutableNodeInputOrOutputMLValue(int index) {
   return const_cast<OrtValue*>(GetNodeInputOrOutputMLValue(index));
 }
 
+Status IExecutionFrame::SetOutputMLValue(int index, const OrtValue& ort_value) {
+  int ort_value_idx = GetNodeIdxToMLValueIdx(index);
+  if (ort_value_idx == NodeIndexInfo::kInvalidEntry || static_cast<size_t>(ort_value_idx) >= all_values_size_) {
+    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "invalid index ", ort_value_idx);
+  }
+
+  all_values_[ort_value_idx] = ort_value;
```

Comment:
>all_values_[ort_value_idx] = ort_val [](start = 2, length = 36)

Is it Okay if the existing value is overwritten?

Reply:
This is a todo item. 
need to properly take care of the overridden ortvalue.